find_the_bug/rectangleArea/solution/rectangleArea.cpp

```diff
@@ -0,0 +1,22 @@
+#include <iostream>
+
+class rectangle {
+    private:
+        double m_base;
+        double m_height;
+    public:
+        rectangle(double base=0., double height=0.) : m_base{base}, m_height{height} {}
```

Comment:
in this case remove the default arguments (and there should be a space before and after the `=` operator)